bool activate
bool new_bag
string filename
----
int8 ERROR=0
int8 SUCCESS=1
int8 NO_CHANGE=2
int8 result
